mappings/net/minecraft/world/dimension/StructuresConfig.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_5311 net/minecraft/world/dimension/StructuresConfig
+	FIELD field_24821 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_24822 DEFAULT_STRUCTURES Lcom/google/common/collect/ImmutableMap;
+	FIELD field_24823 DEFAULT_STRONGHOLD_CONFIG Lnet/minecraft/class_5313;
```

Comment:
```suggestion
	FIELD field_24823 DEFAULT_STRONGHOLD Lnet/minecraft/class_5313;
```

Reply:
I prefer the `_CONFIG` at the end, as this isn't a stronghold but a stronghold config

Reply:
@Kroppeb no, this class is already a config holder, and the actual instance field that holds the stronghold config you've already named `stronghold` below. You need consistency here.